circle.yml

```diff
@@ -2,7 +2,7 @@ machine:
   environment:
     NODE_ENV: test
   node:
-    version: 0.12.6
+    version: 5.1.0
```

Comment:
You are right, should bump. 

I actually stuck with 5.1.0 since that is what I used as a base from nodesource with Docker - they only have up to 5.9.0 so if we have CircleCI support can go up to that one. (Just want one version throughout)